content/programming-cookbook/logging/NLog.md

```diff
@@ -0,0 +1,66 @@
+---
+title: NLog
+showMetadata: true
+editable: true
+showToc: true
+order: 3
```

Comment:
I made a mistake by adding unnecessary order: 3 in `log4net.md`, could you please remove it. We don't need any order in logging content md file.